rest/route/service.go

```diff
@@ -93,12 +93,14 @@ func AttachHandler(app *gimlet.APIApp, opts HandlerOpts) {
 	app.AddRoute("/distros/{distro_id}/execute").Version(2).Post().Wrap(superUser, editDistroSettings).RouteHandler(makeDistroExecute(sc, env))
 
 	app.AddRoute("/hooks/github").Version(2).Post().RouteHandler(makeGithubHooksRoute(sc, opts.APIQueue, opts.GithubSecret, settings))
+	app.AddRoute("/host/range").Version(2).Get().Wrap(checkUser).RouteHandler(makeFetchHostRange(sc))
+	app.AddRoute("/host/start_processes").Version(2).Post().Wrap(checkUser).RouteHandler(makeHostStartProcesses(sc, env))
+	app.AddRoute("/host/get_processes").Version(2).Get().Wrap(checkUser).RouteHandler(makeHostGetProcesses(sc, env))
```

Comment:
do we need more access control?

Reply:
the routes check if the host is owned by the user or a superuser

Reply:
why not check in middleware? 

I guess in the future, we probably want to have more ACL control over this.

Reply:
since the lists of hosts and request parameters (whether specifically `mine`) are part of the body of the request it seemed more appropriate to check in the route.